### PR TITLE
ColorTo Color converters

### PR DESCRIPTION
**Pull request by bijington**
Documentation added for .NET MAUI Community Toolkit proposals:

- [Proposal] `ColorTo Color Converters ` <https://github.com/CommunityToolkit/Maui/issues/52> includes:

  - `ColorToBlackOrWhiteConverter`

  - `ColorToColorForTextConverter`

  - `ColorToGrayScaleColorConverter`

  - `ColorToInverseColorConverter`

### CODE REVIEW

**Comment by PRMerger20** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

File: docs/maui/converters/color-to-inverse-color-converter.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+### XAML
+
+The `ColorToInverseColorConverter` can be used as follows in XAML:
+
+```xaml
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
+             x:Class="CommunityToolkit.Maui.Sample.Pages.Converters.ColorToGrayScaleColorConverterPage">
```
Comment (jfversluis) [inline docs/maui/converters/color-to-inverse-color-converter.md:28]:
```suggestion
             x:Class="CommunityToolkit.Maui.Sample.Pages.Converters. ColorToInverseColorConverterPage">
```

File: docs/maui/converters/color-to-inverse-color-converter.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+}
+```
+
+## Examples
+
+You can find an example of this converter in action in the [.NET MAUI Community Toolkit Sample Application](https://github.com/CommunityToolkit/Maui/blob/main/samples/CommunityToolkit.Maui.Sample/Pages/Converters/ColorsConverterPage.xaml).
+
+## API
+
+You can find the source code for `ColorToGrayScaleColorConverter` over on the [.NET MAUI Community Toolkit GitHub repository](https://github.com/CommunityToolkit/Maui/blob/main/src/CommunityToolkit.Maui/Converters/ColorToColorConverters.shared.cs).
```
Comment (jfversluis) [inline docs/maui/converters/color-to-inverse-color-converter.md:90]:
```suggestion
You can find the source code for `ColorToInverseColorConverter` over on the [.NET MAUI Community Toolkit GitHub repository](https://github.com/CommunityToolkit/Maui/blob/main/src/CommunityToolkit.Maui/Converters/ColorToColorConverters.shared.cs).
```

**Review by jfversluis — CHANGES_REQUESTED** [review]
Some minor copy/paste things!

**Comment by PRMerger5** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by PRMerger3** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by PRMerger14** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

File: docs/maui/converters/color-to-black-or-white-converter.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
+---
+title: ColorToBlackOrWhiteConverter - .NET MAUI Community Toolkit
+author: bijington
+description: "The ColorToBlackOrWhiteConverter is a one way converter that allows users to convert an incoming `Color` to a monochrome value of either Colors.Black or Colors.White."
```
Comment (jfversluis) [inline docs/maui/converters/color-to-black-or-white-converter.md:4]:
```suggestion
description: "The ColorToBlackOrWhiteConverter is a one way converter that allows users to convert an incoming Color to a monochrome value of either Colors.Black or Colors.White."
```

Comment (bijington) [reply docs/maui/converters/color-to-black-or-white-converter.md:4]: Good catch thanks

**Comment by PRMerger16** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by PRMerger17** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Review by jfversluis — APPROVED** [review]

**Pull request merged**